mesa/visualization/components/matplotlib.py

```diff
@@ -11,7 +11,7 @@
 from matplotlib.figure import Figure
 
 import mesa
-from mesa.experimental.cell_space import VoronoiGrid
+from mesa.experimental.cell_space import OrthogonalMooreGrid, VoronoiGrid
```

Comment:
This should also work for OrthogonalVonNeumannGrid

Reply:
yes, I'll fix that! Just need to figure out how to capture this in the match case statement. 

I guess it even will work for hexgrids (although we might want to visualize these a bit differently with offsets for every odd row.